justfile

```diff
@@ -32,4 +23,15 @@ install_linkerd: && link_clusters
 
 #link created two multiclusters
 link_clusters:
-   cd utils && ./link_multilclusters.sh
\ No newline at end of file
+   cd utils && ./link_multilclusters.sh
+
+export K3D_ORG_DOMAIN = env_var_or_default("K3D_ORG_DOMAIN", "cluster.local")
+export K3D_NETWORK_NAME = env_var_or_default("K3D_NETWORK_NAME")
+export K3D_CLUSTER_NAME = env_var_or_default("K3D_CLUSTER_NAME")
+k3d-create:
```

Comment:
There's no point imo to use a build system if we're going to be simply calling scripts from another directory. It seems more work at this point. Let's be a bit more mindful of what a build system helps us achieve here:

* Modularity (we can glue pieces together to create more complex pipelines)
* Customisation (we can pass different arguments to our build system to test different scenarios)
* Sane defaults

If we separate pieces in our build system, we have some more advantages:
* We don't need to run an entire script whenever something has changed (think of compilation in C when you'd have to re-compile _everything_). If we want to add a new cluster, we can just do `just k3d-create` followed by `just install` and `just link`. We can't do that with a script.

* Automation: will be hard to do in CI based off just the scripts. It's hard to maintain.